code/modules/mob/mob.dm

```diff
@@ -1001,6 +1001,8 @@ var/list/slot_equipment_priority = list( \
 /mob/proc/show_stat_station_time()
 	stat(null, "Round Time: [worldtime2text()]")
 	stat(null, "Station Time: [station_time_timestamp()]")
+	if(client.holder && (client.holder.rights & R_ADMIN))
```

Comment:
Kyep suggested this could be check_rights(R_ADMIN). I had put it to this as check_rights does not appear elsewhere in mob.dm, so I'm not sure how it'll work.

Reply:
I believe check_rights relies on usr when the mob isn’t directly passed, which probably doesn’t work in mob procs. 

If you pass src as the third argument it should work.

Reply:
It may be better to just include this on the MC tab's info instead of here.